audit.rdf

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+
+<rdf:RDF xmlns:owl="http://www.w3.org/2002/07/owl#"
+         xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+         xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+         xmlns:skos="http://www.w3.org/2004/02/skos/core#">
+
+  <owl:Ontology rdf:about="http://fedora.info/definitions/v4/audit#">
+    <rdfs:label xml:lang="en">Fedora Commons Repository - Audit Types</rdfs:label>
+    <owl:versionInfo>v4/2015/04/07</owl:versionInfo>
+    <owl:priorVersion rdf:resource=""/>
+  </owl:Ontology>
+
+  <owl:Class rdf:about="http://fedora.info/definitions/v4/audit#InternalEvent">
+    <rdfs:label xml:lang="en">internal event</rdfs:label>
+    <rdfs:comment xml:lang="en">An event generated by the Fedora Repository.</rdfs:comment>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://fedora.info/definitions/v4/audit#ExternalEvent">
+    <rdfs:label xml:lang="en">external event</rdfs:label>
+    <rdfs:comment xml:lang="en">An event generated by an external process and optionally registered
+        with the Fedora Repository.</rdfs:comment>
+  </owl:Class>
+
+  <owl:NamedIndividual rdf:about="http://fedora.info/definitions/v4/audit#contentModification">
+    <rdfs:label xml:lang="en">content modification</rdfs:label>
+    <skos:prefLabel xml:lang="en">content modification</skos:prefLabel>
+    <rdfs:comment xml:lang="en">A content file is modified.</rdfs:comment>
+    <skos:definition xml:lang="en">A content file is modified.</skos:definition>
+    <skos:inScheme rdf:resource="http://id.loc.gov/vocabulary/preservation/eventType"/>
+    <skos:inScheme rdf:resource="http://id.loc.gov/vocabulary/preservation"/>
+    <rdf:type rdf:resource="http://www.w3.org/2004/02/skos/core#Concept"/>
+  </owl:NamedIndividual>
+
+  <owl:NamedIndividual rdf:about="http://fedora.info/definitions/v4/audit#derivativeCreation">
```

Comment:
Yes, that's the pattern I would normally expect -- though there may be some people using sequencers to effectively make these internal events.